src/App.tsx

```diff
@@ -10,33 +12,26 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTab, setSelectedTab] = useState<Tab>(tabs[0]);
+
+  const onTabSelected = (choisedTab: Tab) => {
+    const [currentTab] = tabs.filter(tab => tab.id === choisedTab.id);
```

Comment:
```suggestion
    const currentTab = tabs.find(tab => tab.id === choisedTab.id);
```
Only one tab can be selected

Reply:
done